kubeflow/pipelines.py

```diff
@@ -425,14 +430,15 @@ def train_on_synthetic_dataset_unity_simulation(
     config: str = "datasetinsights/configs/faster_rcnn_synthetic.yaml",
     tb_log_dir: str = "gs://<bucket>/runs/yyyymmdd-hhmm",
     checkpoint_dir: str = "gs://<bucket>/checkpoints/yyyymmdd-hhmm",
-    volume_size: str = "100Gi",
+    volume_size: str = "1.2Ti",
```

Comment:
Why change the default value here? This value can be changed by user when they are running kubeflow pipeline, right? Should we update the tutorial accordingly to let user know this value has to be increased according to their dataset size?

Reply:
I will sugget keeping default value so that user does not hit with huge resource budget. In the tutorial, it should be clear they need to increase this value if they are working on the dataset.